packages/wasmfs/lib/index.ts

```diff
@@ -1,6 +1,7 @@
 // An alternative fs for the browser and testing
 import { createFsFromVolume, IFs } from "memfs";
 import { Volume } from "memfs/lib/volume";
+import "./node_sync_emit";
```

Comment:
I see this file is imported, but not committed? 🤔

Reply:
It's already present on the main source (I just import it to make sure `emit` is synchronous).

Reply:
Sounds good 😄